tests/integration/test_logcollector/test_command_monitoring/test_command_execution_freq.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+import sys
+from datetime import timedelta, datetime
+
+from wazuh_testing import logger
+from wazuh_testing.tools import get_service, monitoring
+from wazuh_testing.tools.time import TimeMachine
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, AGENT_DETECTOR_PREFIX
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_command_conf.yaml')
+wazuh_component = get_service()
+
+if sys.platform == 'win32':
+    prefix = AGENT_DETECTOR_PREFIX
+else:
+    prefix = LOG_COLLECTOR_DETECTOR_PREFIX
+
+local_internal_options = {
+    'logcollector.remote_commands': 1,
+    'logcollector.debug': 2,
+    'monitord.rotate_log': 0
+}
+
+parameters = [
+    {'LOG_FORMAT': 'command', 'COMMAND': 'echo execution frequency 300', 'FREQUENCY': 300},  # 5 minutes.
+    {'LOG_FORMAT': 'command', 'COMMAND': 'echo execution frequency 1800', 'FREQUENCY': 1800},  # 30 minutes.
+    {'LOG_FORMAT': 'command', 'COMMAND': 'echo execution frequency 3600', 'FREQUENCY': 3600},  # 1 hour.
+    {'LOG_FORMAT': 'command', 'COMMAND': 'echo execution frequency 86400', 'FREQUENCY': 86400},  # 24 hours.
+    {'LOG_FORMAT': 'full_command', 'COMMAND': 'echo full_command frequency 300', 'FREQUENCY': 300},  # 5 minutes.
+    {'LOG_FORMAT': 'full_command', 'COMMAND': 'echo full_command frequency 1800', 'FREQUENCY': 1800},  # 30 minutes.
+    {'LOG_FORMAT': 'full_command', 'COMMAND': 'echo full_command frequency 3600', 'FREQUENCY': 3600},  # 1 hour.
+    {'LOG_FORMAT': 'full_command', 'COMMAND': 'echo full_command frequency 86400', 'FREQUENCY': 86400}  # 24 hours.
+]
+metadata = [
+    {'log_format': 'command', 'command': 'echo execution frequency 300', 'frequency': 300},  # 5 minutes.
+    {'log_format': 'command', 'command': 'echo execution frequency 1800', 'frequency': 1800},  # 30 minutes.
+    {'log_format': 'command', 'command': 'echo execution frequency 3600', 'frequency': 3600},  # 1 hour.
+    {'log_format': 'command', 'command': 'echo execution frequency 86400', 'frequency': 86400},  # 24 hours.
+    {'log_format': 'full_command', 'command': 'echo full_command frequency 300', 'frequency': 300},  # 5 minutes.
+    {'log_format': 'full_command', 'command': 'echo full_command frequency 1800', 'frequency': 1800},  # 30 minutes.
+    {'log_format': 'full_command', 'command': 'echo full_command frequency 3600', 'frequency': 3600},  # 1 hour.
+    {'log_format': 'full_command', 'command': 'echo full_command frequency 86400', 'frequency': 86400}  # 24 hours.
+]
+
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOG_FORMAT'], x['COMMAND'], x['FREQUENCY']}" for x in parameters]
```

Comment:
As in the other test, I think that here the command is not relevant and I would love to have a single string instead of a list as an id, for example: "command_5m", "full_command_24h", "command_1h" maybe would be better than these automatic ids. Please, consider adding an extra field to the metadata with the time in human-readable format (minutes, hours, days..) and use it to build something like the following code instead:

```
configuration_ids = [f"{x['log_format']}_{x['freq_str']}" for x in metadata]
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1257/commits/1e33fb8c458aeb8a738e22706713ec98fbf25703